### PR TITLE
Bonifacy

### PR DESCRIPTION
**Pull request by Rogal255**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @Rogal255

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Rogal255 for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
-    return 0;
+
+    if(sequence == 0)
+    {
+        return 0;
+    }
+    else if(sequence == 1)
+    {
+        return 1;
+    }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:12]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
-    return 0;
+        
+    if(sequence == 0)
+    {
+        return 0;
+    }
+    else if(sequence == 1)
+    {
+        return 1;
+    }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:35]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
+
+    int i {0};
+    int j {1};
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:15]: What are 'i' and 'j' ? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**